src/app/lib/utilities/ichefURL/index.js

```diff
@@ -1,5 +1,13 @@
 import { getEnvConfig } from '../getEnvConfig';
 
+const webpSupportedPatterns = [
```

Comment:
Would this mean that according to '/\/ace\/(?:standard|ws)\/.+(?:\/amz\/worldservice\/)?.*/', if the url has ace/standard and amz/worldservice then it will always have .webp added to the end, even if it already has. webp on the end? Do some of the ace/standard and amz/worldservice urls ever already have .webp on the end? Just because if we add .webp to one that already has .webp on the end it makes a bad url. Otherwise this looks good! But just checking that scenario. I see the other IRL in this array doesn't have that problem because it specifies .jpg or .png on the end.

Reply:
Just spoke to @eagerterrier and the URLs won't already have webp on. It is only added in this buildIChefUrl file and the service worker. So this should be ok!

Reply:
Brill. 
I've added [this](https://github.com/bbc/simorgh/pull/11595/files#diff-3fa5355da1b796a6f42917cff3ccfec25c270548c0834cf1462eda73a3e92cceR53) test to ensure we're not appending `.webp` if it's already in the URL.

Reply:
As regards [this](https://github.com/bbc/simorgh/pull/11595/files#r1592204915) query, since the matching pattern checks that the URL ends with `.jpg` or `.png` as per `(?:\.jpg|\.png)`, we wouldn't append `.webp` to the URL if already exists.
This is because [this](https://github.com/bbc/simorgh/pull/11595/files#diff-52550de5c2974525a080b5738d3ac102090a1c226f660df5183015286b2129bbR4) pattern will return `false` when we check for a match.